src/app/workspace/results/results.component.ts

```diff
@@ -29,11 +28,9 @@ export class ResultsComponent {
     searchMode: 'fulltext' | 'gravsearch';
 
     // progress status
-    loading = true;
+    loading = false;
```

Comment:
is this still needed? I don't think this variable is used anywhere now.

Reply:
wow, well seen 👍 no it's not needed anymore.

Removed in ed38403